packages/fast-usdc/test/cli/airdrop-commands.test.js

```diff
@@ -0,0 +1,245 @@
+const handleFetchNetworkConfig = ({ env, fetch }) =>
+  fetchEnvNetworkConfig({ env, fetch });
+
+const handleMakeSmartWalletKit =
+  ({ delay, fetch }) =>
+  networkConfig =>
+    makeSmartWalletKit({ delay, fetch }, networkConfig);
+
+const makeClaimAirdropOffer = (
+  { feeBrand, instanceName },
+  { proof, tier, pubkey },
+) => ({
+  id: `offer-${pubkey.slice(0, 10)}`,
+  invitationSpec: {
+    source: 'agoricContract',
+    instancePath: [instanceName],
+    callPipe: [['makeClaimTokensInvitation']],
+  },
+  proposal: {
+    give: {
+      Fee: {
+        brand: feeBrand,
+        value: 5n,
+      },
+    },
+  },
+  offerArgs: {
+    proof,
+    tier,
+    pubkey,
+  },
+});
+
+const trace = label => value => {
+  console.log(label, '::::', value);
+  return value;
+};
+const traceAsync = label => value => Promise.resolve(value).then(trace(label));
+
+const composeM =
+  method =>
+  (...ms) =>
+    ms.reduce((f, g) => x => g(x)[method](f));
+
+const composePromises = composeM('then');
+
+// Successfully
+const collectPowers = config =>
+  composePromises(
+    trace('inspecting walletKit capabilities'),
+    handleMakeSmartWalletKit(config),
+    traceAsync('network config'),
+    () => handleFetchNetworkConfig({ env: config.env, fetch: config.fetch }),
+  );
+
+const addAirdropCommands = (program, { fetch, stderr, stdout }) => ({
+  operator: program.command('claimer').description('Claim airdrop commands'),
+  invoke() {
+    this.operator
+      .command('send')
+      .description(
+        'constructs an OfferSpec in accordance with makeClaimTokensInvitation from ertp-airdrop.',
+      )
+      .option('--proof <string>', 'Proof', String)
+      .option('--pubkey <string>', 'Public key', String)
+      .option('--tier <number>', 'Tier', Number)
+      .action(async opts => {
+        const { proof, tier, pubkey } = opts;
+        const powers = await collectPowers({
+          delay: 500,
+          fetch,
+          env: { AGORIC_NET: 'xnet' },
+        })();
+
+        const [instance, brand] = await getInstanceAndBrand({
+          brandName: 'IST',
+          instanceName: 'tribblesXnetDeployment',
+        })(powers);
+        console.log('------------------------');
+        console.log({ instance, brand });
+
+        /** @type {import('@agoric/smart-wallet/src/invitations.js').InvitationSpec} */
+        const offer = makeClaimAirdropOffer(
+          { feeBrand: brand, instanceName: 'tribblesXnetDeployment' },
+          { proof, tier, pubkey },
+        );
+
+        console.log({ offer });
+        outputActionAndHint(
+          { method: 'executeOffer', offer },
+          { stderr, stdout },
+          powers.marshaller,
+        );
+      });
+    return this.operator;
+  },
+});
+
+export const delay = ms => new Promise(resolve => setTimeout(resolve, ms));
+
+// Alternative: Async pipe executes functions left to right (sometimes more intuitive)
+
+test('tribbles airdrop program', async t => {
```

Comment:
Steps for running:
		
1. pull down `tg-cli-extension`
2. cd into `agoric-sdk/packages/fast-usdc`
3. execute the following command
```
yarn ava ./test/cli/airdrop-commands.test.js
```